walkthroughs/week-5-tdd/project/src/main/webapp/index.html

```diff
@@ -20,6 +20,10 @@ <h2>Attendees</h2>
       <p>Who needs to attend the meeting (comma-separated list)?</p>
       <input id="attendees" type="text" placeholder="Amelia, Ava, Emma" />
 
+      <h2>Optional Attendees</h2>
+      <p>Who can attend the meeting optionally (comma-separated list)?</p>
+      <input id="optional-attendees" type="text" placeholder="Amelia, Ava, Emma" />
```

Comment:
probably should have different names for the optional-attendees than the attendees, even though it's just a placeholder

Reply:
Done!